README.md

```diff
@@ -1,6 +1,9 @@
 # ocis-php-sdk
 This SDK allows you to interact with [ownCloud Infinite Scale (oCIS)](https://github.com/owncloud/ocis/) storage using PHP.
 
+## Documentation
+You can find a rendered version of the [API documentation](https://owncloud.dev/ocis-php-sdk/) in our dev docs.
+
```

Comment:
Is there any documentation to locally serve documentation

Reply:
good point, added some description in 696f213